packages/manager/modules/pci/src/projects/project/kubernetes/components/autoscaling/autoscaling.controller.js

```diff
@@ -0,0 +1,140 @@
+import { AUTOSCALING_LINK } from './autoscaling.constants';
+import { SCALE_DEFAULT_VALUES } from '../../kubernetes.constants';
+
+export default class AutoscalingController {
+  /* @ngInject */
+  constructor(coreConfig) {
+    this.ovhSubsidiary = coreConfig.getUser().ovhSubsidiary;
+    this.autoscalingInfoLink = this.getAutoscalingInfoLink();
+  }
+
+  $onInit() {
+    this.setLowestHighestLimit();
+    this.model.isValidScale = AutoscalingController.isValidKubeScale(
+      this.model.autoscale,
+      this.model.nodes.lowest.value,
+      this.model.nodes.desired.value,
+      this.model.nodes.highest.value,
+    );
+  }
+
+  getAutoscalingInfoLink() {
+    return AUTOSCALING_LINK[this.ovhSubsidiary] || AUTOSCALING_LINK.DEFAULT;
+  }
+
+  resetScaleValuesToDefault() {
+    const { lowest, highest } = this.model.nodes;
+    lowest.min = SCALE_DEFAULT_VALUES.LOWEST_MIN_VALUE;
+    lowest.value = SCALE_DEFAULT_VALUES.LOWEST_MIN_VALUE;
+
+    highest.value = SCALE_DEFAULT_VALUES.HIGHEST_VALUE;
```

Comment:
above you have set both lowest values to LOWEST_MIN_VALUE. It's different for the highest. Is it ok?

Reply:
Yes it's normal.
If costumer disable the scale option the lowest should be set to 0 but the max to default which is 100 as max.